Translate untranslated chapters to Polish

Unrooted opened the pull request:
This translates the untranslated chapter of the Tour of Scala into Polish.

eed3si9n commented:
Thanks for the contribution! I can't review this myself so could you find someone who can? Fortunately Poland has no shortage of knowledgable Scala programmers so I'm sure you can find someone in no time. /cc @Krever, @romanowski, @rkrzewski, @EncodePanda

Unrooted commented:
I added f6e4b5051b1fb8ac7d0a9a390bb92d8b40fb7fc4 to the #1758 a few days ago and it's also added in here, I guess we can close #1758 and focus on this PR since it also does include the same thing but doesn't have such a mess in logs.

Krever reviewed (changes requested):
Few changes, but looks good in general, thanks! 
I haven't checked the new pages ordering though, I hope its alright.

Krever reviewed (approved):
LGTM. Would be good if one more person could look at it, but if not should still be ok.

rkrzewski commented on _pl/tour/by-name-parameters.md:
```diff
@@ -1,6 +1,41 @@
 language: pl
 ---
+
+_Parametry przekazywane według nazwy_ są ewaluowane za każdym razem gdy są używane. Nie zostaną w ogóle wyewaluowane jeśli nie będą używane. Jest to podobne do zastępowania parametrów według nazwy przekazanymi wyrażeniami. Są przeciwieństwem do _parametrów według wartości_. Aby utworzyć parametr wywoływany według nazwy, po prostu dodaj `=>` przed jego typem.
```
```suggestion
_Parametry przekazywane według nazwy_ są ewaluowane za każdym razem gdy są używane. Nie zostaną w ogóle wyewaluowane jeśli nie będą używane. Jest to podobne do zastępowania parametrów w ciele funkcji wyrażeniami podanymi w miejscu jej wywołania. Są przeciwieństwem do _parametrów przekazywanych według wartości_. Aby utworzyć parametr przekazywany według nazwy, po prostu dodaj `=>` przed jego typem.
```

rkrzewski commented on _pl/tour/by-name-parameters.md:
```diff
@@ -1,6 +1,41 @@
+
+Metoda `whileLoop` używa wielu list parametrów do określenia warunku i treści pętli. Jeśli `condition` (warunek) jest prawdziwy, `body` (treść) jest wykonywana a następnie wykonywane jest rekurencyjne wywołanie `whileLoop`. Jeśli `condition` jest fałszywy, treść nigdy nie jest ewaluowana, ponieważ dodaliśmy `=>` do typu `body`.
+
+Teraz, kiedy przekażemy `i>0` jako nasz `condition` (warunek) i `println(i); i-= 1` jako `body`, nasze wyrażenie zachowuje się jak standardowa pętla while w wielu językach.
```
```suggestion
Jeśli przekażemy `i>0` jako nasz `condition` (warunek) i `println(i); i-= 1` jako `body`, nasze wyrażenie zachowuje się jak standardowa pętla while w wielu językach.
```

rkrzewski commented on _pl/tour/by-name-parameters.md:
```diff
@@ -1,6 +1,41 @@
+
+Teraz, kiedy przekażemy `i>0` jako nasz `condition` (warunek) i `println(i); i-= 1` jako `body`, nasze wyrażenie zachowuje się jak standardowa pętla while w wielu językach.
+
+Ta możliwość opóźnienia ewaluacji parametru do czasu jego użycia może zwiększyć wydajność, jeśli ewaluacja parametru wymaga intensywnych obliczeń lub dłużej działającego bloku kodu, takiego jak pobieranie adresu URL.
```
```suggestion
Ta możliwość opóźnienia ewaluacji parametru do czasu jego użycia może zwiększyć wydajność, jeśli ewaluacja parametru wymaga intensywnych obliczeń lub dłużej działającego bloku kodu, takiego jak pobieranie treści spod adresu URL.
```

rkrzewski commented on _pl/tour/package-objects.md:
```diff
@@ -1,14 +1,68 @@
+
+Scala udostępnia obiekty pakietu jako wygodny kontener wspóldzielony w całym pakiecie.
+
+Obiekty pakietu mogą zawierać dowolne definicje, nie tylko definicje zmiennych i metod. Na przykład, są często używane do przechowywania aliasów typów dla całego pakietu i niejawnych konwersji. Obiekty pakietu mogą nawet dziedziczyć klasy i cechy (traits) Scali.
```
```suggestion
Obiekty pakietu mogą zawierać dowolne definicje, nie tylko definicje zmiennych i metod. Na przykład, są często używane do przechowywania aliasów typów i niejawnych konwersji. Obiekty pakietu mogą nawet dziedziczyć klasy i cechy (traits) Scali.
```

rkrzewski commented on _pl/tour/package-objects.md:
```diff
@@ -1,14 +1,68 @@
+
+Obiekty pakietu mogą zawierać dowolne definicje, nie tylko definicje zmiennych i metod. Na przykład, są często używane do przechowywania aliasów typów dla całego pakietu i niejawnych konwersji. Obiekty pakietu mogą nawet dziedziczyć klasy i cechy (traits) Scali.
+
+Z założenia, kod źródłowy obiektu pakietu jest zwykle umieszczany w pliku źródłowym o nazwie `package.scala`.
```
```suggestion
Zgodnie z konwencją, kod źródłowy obiektu pakietu jest zwykle umieszczany w pliku źródłowym o nazwie `package.scala`.
```

rkrzewski commented on _pl/tour/package-objects.md:
````diff
@@ -1,14 +1,68 @@
+```
+
+Teraz załóżmy, że chcesz umieścić zmienną `planted` i metodę `showFruit` bezpośrednio w pakiecie `gardening.fruits`.
+Oto jak to się robi:
````
```suggestion
Możesz zrobić to w następujący sposób:
```

rkrzewski commented on _pl/tour/package-objects.md:
````diff
@@ -1,14 +1,68 @@
+}
+```
+
+Jako przykład tego, jak wygląda użytkowanie, obiekt `PrintPlanted` importuje `planted` i `showFruit` w ten sam sposób, w jaki importuje klasę `Fruit` - używając importu wieloznacznego w pakiecie `gardening.fruits`.
````
```suggestion
Jako przykład tego, jak wygląda użycie definicji przygotowanych w ten sposób, obiekt `PrintPlanted` importuje `planted` i `showFruit` w ten sam sposób, w jaki importuje klasę `Fruit` - używając importu wieloznacznego w pakiecie `gardening.fruits`.
```

rkrzewski commented on _pl/tour/packages-and-imports.md:
```diff
@@ -1,15 +1,92 @@
+
+## Import
+
+Deklaracje `import` służą do uzyskiwania dostępu do członków (members, tzn.: klasy, cechy, funkcje itp.) w innych pakietach. Aby uzyskać dostęp do członków tego samego pakietu, nie jest wymagana deklaracja `import`. Deklaracje `import` są selektywne.
```
```suggestion
Deklaracje `import` służą do uzyskiwania dostępu do elementów składowych (members, tzn.: klasy, cechy, funkcje itp.) w innych pakietach. Aby uzyskać dostęp do elementów tego samego pakietu, nie jest wymagana deklaracja `import`. Deklaracje `import` są selektywne.
```

rkrzewski commented on _pl/tour/packages-and-imports.md:
````diff
@@ -1,15 +1,92 @@
+```
+import users._  // zaimportuj wszystko z pakietu użytkowników
+import users.User  // zaimportuj klasę User
+import users.{User, UserPreferences}  // zaimportuj tylko wybranych członków
````
```suggestion
import users.{User, UserPreferences}  // zaimportuj tylko wybrane elementy
```

rkrzewski commented on _pl/tour/packages-and-imports.md:
````diff
@@ -1,15 +1,92 @@
+import users.{UserPreferences => UPrefs}  // zaimportuj i zmień nazwę dla wygody
+```
+
+Jedną z różnic w Scali od Javy jest to, że `import` można używać wszędzie:
````
```suggestion
Jedną z różnic w Scali od Javy jest to, że deklarację `import` można umieścić w dowolnym miejscu:
```

rkrzewski commented on _pl/tour/packages-and-imports.md:
````diff
@@ -1,15 +1,92 @@
+}
+```
+
+W przypadku konfliktu nazw i konieczności zaimportowania czegoś z katalogu głównego projektu, poprzedź nazwę pakietu przedrostkiem `_root_`:
````
```suggestion
W przypadku konfliktu nazw i konieczności podania pełnej ścieżki w hierarchii nazw pakietów, poprzedź nazwę pakietu przedrostkiem `_root_`:
```

rkrzewski commented on _pl/cheatsheets/index.md:
```diff
@@ -1,88 +1,627 @@
-|  `x.isInstanceOf[String]`                                                                                |  sprawdzenie typu (w czasie wykonania) |
-|  `x.asInstanceOf[String]`                                                                                |  rzutowanie typu (w czasie wykonania) |
-|  `x: String`                                                                                             |  oznaczenie typu (w czasie kompilacji) |
+<table>
```
Markdown code was replaced by HTML here. Was that intended?

rkrzewski reviewed (commented):
Hey, I've added some edit suggestions for the consideration of translation's author.

Unrooted commented:
Yes, it was intended. I did it to make it consistent with the English version of the cheat sheet.

SethTisue commented:
Is this ready for merge?

Unrooted commented:
Yes, it is. I added all requested changes, it is ready for merge.

SethTisue commented:
thank you, @Unrooted and reviewers, for this substantial contribution!
